package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.jsx",
   "scripts": {
     "build": "webpack",
-    "start": "webpack-dev-server --open"
+    "start": "webpack-dev-server"
```

Comment:
Required to fix the following error (displayed when browse to `http://localhost:8080/`):
```
Cannot GET /
```